DigitalLearningSolutions.Data/Services/LoginService.cs

```diff
@@ -156,12 +156,12 @@ public LoginResult AttemptLogin(string username, string password)
 
         private static bool MultipleEmailsUsedAcrossAccounts(AdminUser? adminUser, List<DelegateUser> delegateUsers)
         {
-            var emails = delegateUsers.Select(du => du.EmailAddress)
+            var emails = delegateUsers.Select(du => du.EmailAddress?.ToLower())

```

Comment:
Should we be using `ToLowerInvariant` or `Tolower` with a specified culture? We could conceivably see an issue with this if a person's locale is set to certain cultures ([StackOverflow example](https://stackoverflow.com/questions/6600954/c-tolower-is-sometimes-removing-dot-from-the-letter-i)).

Reply:
I don't think it will actually change anything for us, assuming my understanding of how the culture stuff work is correct. I thought it was based of the device the application was running on, in this case the server that we (HEE) control and is therefore most likely to be English (UK) or English (US). It sounds like the example in that StackOverflow is some kind of client app (which could then be picking up any culture) calling a web service and failing. I've changed it anyway, because I don't think it makes any difference is actually correctly set to English.

Reply:
Yeah, I feel like it's better to be safe than sorry here since if our locale is fixed to English there will be no difference between `ToLower` and `ToLowerInvariant` but if it isn't we're avoiding a hard to track down bug. Thanks for changing it 🙂